about.html

```diff
@@ -0,0 +1,185 @@
+        <div class="about-desc">
+          <p>
+            part
+            in
+            finishing the task. The Theme Verse for the conference will be Romans 10:14 (...how are they to believe in
+            him
+            of whom they have never heard?).</p>
+        </div>
+        <div class="contact-us">
+          <p>Please contact us per email incase of any inquiris. you are also <br> welcome to visit our offices</p>
+          <p class="email">info@ekklesiaafrika.org | Tel: +254 746 199 225 </p>
+        </div>
+      </section>
+      <section class="middle">
+         <div class="theme">
+           <h3>The Missions Logo</h3>
+           <p>This years Logo was decided after seeing the misunderstanding especially in the african church.We believe
+             that missions </p>
+         </div>
+         <div>
+           <div class="loggo">
+             <img src="./images/logo.webp" alt="logo">
+           </div>
+         </div>
+      </section>
+    </section>
+     <section class="mobile">
+
+       <div class="mobile-menu">
+         <a href="index.html" class="mobile-item">Home
+         </a>
+         <a href="about.html" class="mobile-item">About
+         </a>
+         <a href="#" class="mobile-item">Program
+         </a>
+         <a href="#" class="mobile-item">Join
+         </a>
+         <a href="#" class="mobile-item">Sponsor
+         </a>
+
+       </div>
+       <div>
+         <span class="icon-div">
+           <i class="fa fa-window-close fa-3x icon closeButton"></i>
+         </span>
+       </div>
+     </section>
+    <section class="past-events">
+      <div class="past-conf">
+        <h2>Past Conferences</h2>
+        <hr class="line">
+        <p>This being our eighth, here is a snippet of our past conferences</p>
+      </div>
+      <div class="events">
+        <div class="event-item">
+          <div class="event-desc">
+            <div>
+              <h2 class="event">2019</h2>
+              <p>Mamlak HIll chapel</p>
+            </div>
+
+          </div>
+        </div>
+        <div class="event-item">
+          <div class="event-desc">
+            <div>
+              <h2 class="event">2018</h2>
+              <p>GracePOint Kikuyu</p>
+            </div>
+
+          </div>
+        </div>
+      </div>
+    </section>
+    <section class="partners partners-about">
+      <h2 class="partner-title">partners</h2>
+      <hr class="line">
+      <ul class="partner-list">
+        <li class="partner">
+          <h2>EKKLESIA</h2>
+        </li>
+        <li class="partner">
+          <h2>BIBLICA</h2>
+        </li>
+        <li class="partner">
+          <H2>RBC</H2>
+        </li>
+
+      </ul>
+      <ul class="partner-list">
+        <li class="partner-row">
+          <h2>HOPECITY</h2>
+        </li>
+        <li class="partner-row">
+          <h2>MAMLAKA</h2>
+        </li>
+      </ul>
+    </section>
```

Comment:
- [ ] According to [design guidelines](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015) your partner section could improve for the next points (according to the last review, some of them are optional because the last reviewer doesn't tell it, but I invite you to consider too):
- The partners could be centered and as users, we could see the partners list with all names complete (consider this, the partners in some cases are the organizations who gave money, so, it's important to show them as appropriate).
- [OPTIONAL] Margin top for the title section
- [OPTIONAL] Title section font, it looks like the title doesn't have an appropriate font family, kindly work on that.
- [OPTIONAL] According to the example, the background color is #414246. Why not change it to be more according to the design template?

| given design | your design |
| --- | --- |
| ![image](https://user-images.githubusercontent.com/20563252/128178436-48536ef9-220f-4e96-bff5-d29de9dcc2d7.png) | ![image](https://user-images.githubusercontent.com/20563252/128178528-4bb85604-9068-4904-9db5-3e153b3de1ec.png) |

Reply:
Hello Victor, thanks for the recommendations, However it is quite unfair that you would come up with a new recommendation after I have worked on all the changes you requested. I think it would be prudent to request all the changes at once as the number of reviews are limited.